pym/_emerge/resolver/output.py

```diff
@@ -699,22 +705,28 @@ def do_changelog(self, pkg, pkg_info):
 		return
 
 
-	def check_system_world(self, pkg):
-		"""Checks for any occurances of the package in the system or world sets
+	def check_sets(self, pkg):
+		"""Checks for any occurances of the package in the portage sets
 
 		@param pkg: _emerge.Package.Package instance
-		@rtype system and world booleans
+		@rtype set, system, and world booleans
 		"""
 		root_config = self.conf.roots[pkg.root]
-		system_set = root_config.sets["system"]
-		world_set  = root_config.sets["selected"]
-		system = False
-		world = False
+		set = None
```

Comment:
`set` is a built-in class in Python. Do not name local variables in such way. I suggest `user_set`.